src/core/currency/wallet/currency-wallet-api.js

```diff
@@ -17,6 +17,7 @@ import {
   type EdgeGetTransactionsOptions,
   type EdgeMetadata,
   type EdgeParsedUri,
+  type EdgeParseUriOptions,
```

Comment:
Make this a cleaner

Reply:
Per our discussion, a cleaner is not needed. The other code should simply trust the Flow / Typescript system to do it's job, and not validate stuff it doesn't need to.